YoCode/FileImport.cs

```diff
@@ -13,17 +13,22 @@ namespace YoCode
 
     public class FileImport
     {
-        public string ORIGINAL_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\junior-test";
-        public string MODIFIED_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\original-test";
+        // This shouldn't live in FileImport class
+
+        public string ORIGINAL_PATH { get; set; } = @"..\..\..\..\..\junior-test";
+
+        public string MODIFIED_PATH { get; set; } = @"..\..\..\..\..\original-test";
+
+
 
         List<String> SearchPatterns = new List<string>{ "*.cs", "*.cshtml", "*.js" };
         
         //Will return a list of all files from a directory
         public List<string> GetAllFilesInDirectory(String PATH)
```

Comment:
ALLCAPS is definitely not normal for parameters, this should just be called `path`.